src/app/ui/courses/courses.component.ts

```diff
@@ -10,12 +10,12 @@ import {combineLatest, zip} from 'rxjs';
   styleUrls: ['./courses.component.css'],
 })
 export class CoursesComponent implements OnInit {
-
   courses: Course[];
   term: any;
   route: ActivatedRoute;
   router: Router;
 
+
```

Comment:
Please clean these unnecessary changes, we want to keep our commits as clean as possible :)